axon-reactor/src/main/java/org/axonframework/extensions/reactor/commandhandling/gateway/DefaultReactorCommandGateway.java

```diff
@@ -126,6 +124,10 @@ public Registration registerResultHandlerInterceptor(
                 .map(it -> (R) it.getPayload());
     }
 
+    private Mono<? extends CommandResultMessage<?>> mapExceptionalResult(CommandResultMessage<?> response) {
```

Comment:
Nit: I'd rename the parameter to `result` instead of `response`, to more closely align with the type.